.gitignore

```diff
@@ -40,6 +40,7 @@ out/
 /build-support/virtualenv-*
 /build-support/virtualenv.dist/
 /build-support/phab/
+build-support/twine-deps.venv/
```

Comment:
Nit: the convention is to prefix with `/` so that this ignore only applies at the root level. Would be good to do for consistency.